tests/software/pairtools/flip/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+process NO_PG {
```

Comment:
Woops. What is this?!

Reply:
This is because the pysam will automatic create a @PG line to include the command information. And it will include absolute path of the command. Where this will cause the different output from Docker and Conda container.

Reply:
Ah, I see. Yeah, not so important to get an md5sum for the SAM file here because it is an input to the module.